### PR TITLE
Removed ts-ignore

### PR DESCRIPTION
**Pull request by arpitBhalla**
**What kind of change does this PR introduce?**
removed ts-ignore in following
- [x] FeatureTile
- [x] withTheme 

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**

**Summary**
Fixing #2798

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by arpitBhalla** [comment]
@flyingcircle  Kindly Review

**Comment by flyingcircle** [comment]
using `as any` kind of defeats the purpose. I'd rather see a proper solution.

**Comment by arpitBhalla** [comment]
@flyingcircle  Please Review

**Review by flyingcircle — CHANGES_REQUESTED** [review]
Your changes to `withTheme` look good, but the changes in `Icon` were off, I fixed the issue here: https://github.com/react-native-elements/react-native-elements/pull/2847/files you can see the issue was a package mismatch in the lock file.

**Comment by arpitBhalla** [comment]
@flyingcircle I have made changes, please review

File: src/icons/Icon.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import {
   StyleProp,
   TextStyle,
   TouchableHighlightProps,
+  RippleBackgroundPropType,
```
Comment (flyingcircle) [inline src/icons/Icon.tsx:12]: This is an unnecessary change. There was another PR that addressed this issue so you can undo all the changes in this file

**Review by flyingcircle — CHANGES_REQUESTED** [review]

File: src/icons/Icon.tsx (hunk 1 above)
Comment (arpitBhalla) [reply src/icons/Icon.tsx:12]: Okay, will do the same

**Comment by arpitBhalla** [comment]
@flyingcircle Kindly review

File: src/icons/Icon.tsx (hunk 1 above)
Comment (arpitBhalla) [reply src/icons/Icon.tsx]: Screenshot from 2021-03-14 10-41-01
